test/e2e/index.ts

```diff
@@ -667,32 +647,40 @@ test(
     });
     const agent = await admin.generateAgentPubKey();
 
+    const zippedDnaBundle = fs.readFileSync("test/e2e/fixture/test.dna");
+
+    const appBundle: AppBundle = {
+      manifest: {
+        manifest_version: "1",
+        name: "app",
+        roles: [
+          {
+            name: role_name,
+            provisioning: {
+              strategy: CellProvisioningStrategy.Create,
+              deferred: false,
+            },
+            dna: {
+              bundled: "dna_1",
+              modifiers: { network_seed: "some_seed" },
+            },
+          },
+        ],
+        membrane_proofs_deferred: true,
```

Comment:
This `membrane_proofs_deferred: true` setting here that deviates from the original manifest is the reason to make bundle the happ with this special manifest explicitly here.